### DISCUSSION TITLE
Aspect Ratio Postprocessing

### DISCUSSION THREAD

**Discussion by alimostafavi24** [idaholab/moose#24008] [Q&A General] [answered]
Hello everyone,

I am solving cahn-hilliard equation and I solved for the order parameter. I am not very familiar with post-processing in MOOSE. The initial condition is the elliptical domain and then the domain evolves and oscillated as it can seen in the attached image. What I need is to get the aspect ratio in each time step and plot it vs time. I would highly appreciate it if someone can tell me how I can get the aspect ratio and plot it vs time.

![Screenshot from 2023-04-08 15-59-59](https://user-images.githubusercontent.com/119982603/230746126-4de39d04-0f97-4c63-9354-88609ac1875d.png)

Thank you,
Ali

**Comment by lindsayad** [comment, marked as answer]
@dschwen do you have any suggestion for the kind of aspect ratio postprocessor this user is asking for? This kind of seems like it would be a nontrivial postprocessor to implement

**Reply by GiudGiud** [reply]
We need some sort of "multi-quantity" postprocessor. Would have been a good base for the PressureDrop PP too
